srcCxx/shape_main.cxx

```diff
@@ -1030,7 +1042,9 @@ class ShapeApplication {
 
                 if (retval == RETCODE_OK) {
                     int i;
-                    for (i = 0; i < samples.length(); i++)  {
+                    logger.log_message("Read " + std::to_string(samples.length())
+                        + " sample(s), printing them...", Verbosity::DEBUG);
+                    for (i = 0; i < samples.length(); i++) {
```

Comment:
This conflicts with the changes just merged in #15

Reply:
fixed